benchmark/CMakeLists.txt

```diff
@@ -18,3 +18,7 @@ raja_add_benchmark(
 raja_add_benchmark(
   NAME ltimes
   SOURCES ltimes.cpp)
+
+raja_add_bench_mark(
```

Comment:
maybe add a guard here to only compile the benchmark for GPU or serial backends?  Or add kernel definitions for OpenMP

Reply:
I just took a pass at the driver, I tried to follow the ltimes version in which we can use define which version to run